### ISSUE TITLE
Bug: JavaScript error on project Gantt view

### ISSUE THREAD

**Issue by rdoursenaud** [Dolibarr/dolibarr#5520]
# Bug
## Environment
- **Version**: 4.0.0-rc
- **URL(s)**: /projet/ganttview.php?id=1

```
2016-07-20 16:47:35.816 jsgantt.js:880 Uncaught TypeError: Cannot read property 'getVisible' of undefinedDrawDependencies @ jsgantt.js:880(anonymous function) @ VM14338:1setTimeout (async)(anonymous function) @ ganttview.php?id=1:256
```

**Comment by olsesacl** [comment]
Fix #5547 

change jsgantt to jsGanttImproved

![tareas](https://cloud.githubusercontent.com/assets/6471493/17121911/39b1ffb2-52d7-11e6-80bc-57e2f377a6c1.jpg)

Colors:
blue if is normal task
green if the task has child
red if task has child and task has parent

Now can colapse group task and it have tooltip

![tarea2](https://cloud.githubusercontent.com/assets/6471493/17122009/1f8079b0-52d8-11e6-9cd2-adc65dbcab53.jpg)

![tarea3](https://cloud.githubusercontent.com/assets/6471493/17122070/ccccfed6-52d8-11e6-8687-44f985c5e7f4.jpg)

**Comment by rdoursenaud** [comment]
Nice :+1:
